catalog/spatial/wfs/spatial-wfs-featuretransformer-handlebars/src/main/java/org/codice/ddf/spatial/ogc/wfs/transformer/handlebars/HandlebarsWfsFeatureTransformer.java

```diff
@@ -230,7 +230,6 @@ private String getNamespaceAlias(String namespace, Map<String, String> namespace
   private String getIdAttributeValue(
       StartElement startElement, Map<String, String> namespaces, String namespaceAlias) {
     String id = null;
-    startElement.getAttributeByName(new QName(namespaces.get(namespaceAlias), "id")).getValue();
```

Comment:
Removed because the `getValue()` call can cause a `NullPointerException` when the element has no `gml:id` attribute. And this statement appears to serve no purpose.